src/components/About.js

```diff
@@ -1,8 +1,34 @@
-import React from "react";
+import React from 'react';
 
 const About = () => {
   return (
-    <h1> Hello This is About page</h1>
+    <main className="mx-5 my-3">
```

Comment:
Instead of `mx-5` we could do `container`?

Reply:
I changed it.